pages/jdl/enums.md

```diff
@@ -24,8 +24,8 @@ sitemap:
 Enumeration declaration is done as follows:
 
 ```
-enum [<enum name>] {
-  <ENUM KEY> ([<enum value>])
+enum <enum name> {
+  <ENUM KEY> [(<enum value>)]
```

Comment:
I don't really see the point here

Reply:
`([<enum value>])` implies that this is valid:
```
enum Foo {
  Bar ()
}
```
And this is invalid
```
enum Foo {
  Bar
}
```

Reply:
Ah very good point! Your eyes are better than mine